packages/manager/modules/billing/src/translations/Messages_fr_FR.json

```diff
@@ -908,5 +908,11 @@
   "statements_operation_description_refund_unknown": "Remboursement",
   "billing_date_range_choose_custom_date": "Définir une période",
   "billing_date_range_text": "Du {{t0}} au {{t1}}",
-  "billing_price_ttc_label": "{{price}} TTC"
+  "billing_price_ttc_label": "{{price}} TTC",
+  "billing_guides_header": "Guides",
```

Comment:
Question: Will Guides needs translation ?

Reply:
Yes, for example in spanish "guides" is said "Guías".